components/EditForm/index.js

```diff
@@ -115,6 +113,7 @@ export default function EditForm({ project }) {
       <Timer project={project} />
       <form onSubmit={handleSubmit}>
         <input
+          aria-label="project name"
```

Comment:
It is an a11y concern and [requirement to add label elements.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#labels_2). Please don't use placeholder text as replacement for a label, because even people without a11y issues will have problems to figure out what they need to put into this field.

If you don't want to display the `<label>` element, use a `.screen-reader-only` class to make it invisible.

On the other side, the mdn description of `aria-label` states: "If none of these options are available, or if the default accessible name is not appropriate, use the aria-label attribute to define the accessible name of an element."

So it's fine for a screen reader.

However, I (and some other coaches) find it hard to figure out where to write down what: this is where labels come in handy.

On a mobile phone, it's too small to click on the correct input field.